deployer/src/util.ts

```diff
@@ -1373,3 +1384,21 @@ export function renamePackage(spec: DeployStructure, oldName: string, newName: s
   }
   return spec
 }
+
+// Checks if a given pattern matches exclusion list, defined by system or user via global .exclude file.
+export function isExcluded(match: string): boolean {
+  return anymatch(getExclusionList(), match)
+}
+
+// Returns full list of exclusion patterns, predefined or listed in the global .exclude file.
+export function getExclusionList(): string[] {
+  let userDefinedPatterns = []
+  try {
+    const globalExcludeFile = path.join(process.cwd(), '.exclude')
```

Comment:
I'm concerned about the use of process.cwd() here.   I would expect to read this information either (1) from the root of the project, or (2) from the user's home directory, depending on what scope you are looking for.   I'm not sure which of those two scopes is better (let's discuss) but I'm pretty sure the current directory is not going to work in general.

Reply:
Thanks!
The project path is fed to the command, using args, so  `process.cwd()` is not the right choice here.
Agree. I think, option 1  to be the right choice.